builds/e2e/e2e.yaml

```diff
@@ -63,26 +63,50 @@ jobs:
     - template: templates/e2e-setup.yaml
     - template: templates/e2e-run.yaml
 
-################################################################################		
-  - job: centos7_amd64	
-################################################################################		
-    displayName: CentOs7 amd64	
-
-    pool:	
-      name: $(pool.name)	
-      demands:	
-        - Agent.OS -equals Linux	
-        - Agent.OSArchitecture -equals X64	
-        - run-new-e2e-tests -equals true	
-
-    variables:	
-      os: linux	
-      arch: amd64	
-      artifactName: iotedged-centos7-amd64	
-
-    steps:	
-    - template: templates/e2e-clean-directory.yaml	
-    - template: templates/e2e-setup.yaml	
-    - template: templates/e2e-clear-docker-cached-images.yaml	
+################################################################################
+  - job: centos7_amd64
+################################################################################
+    displayName: CentOs7 amd64
+
+    pool:
+      name: $(pool.name)
+      demands:
+        - Agent.OS -equals Linux
+        - Agent.OSArchitecture -equals X64
+        - run-new-e2e-tests -equals true
+
+    variables:
+      os: linux
+      arch: amd64
+      artifactName: iotedged-centos7-amd64
+
+    steps:
+    - template: templates/e2e-clean-directory.yaml
+    - template: templates/e2e-setup.yaml
+    - template: templates/e2e-clear-docker-cached-images.yaml
     - template: templates/e2e-run.yaml
 
+################################################################################
+  - job: linux_amd64_proxy
+################################################################################
+    displayName: Linux amd64 behind a proxy
+
+    pool:
+      name: $(pool.name)
```

Comment:
Do you have a walk-through document that goes over the steps to create/maintenance a proxy-test-agent?